using/legal/data-protection-policy.md

```diff
@@ -0,0 +1,33 @@
+# Data Protection Policy
+
+## Overview
+
+The GDPR (General Data Protection Regulation) is UK legislation that governs how we process data originating from users from the UK.
+Exercism aims to be GDPR compliant for all users, even those outside the EU and you therefore have some important rights applicable to our Platform.
+
+1. **Your Right to be Informed:**
+   This Privacy Policy details who we are, what we collect, and what we use it for.
+2. **Right of access:**
+   At any time you can review the information on your profile and update or change it.
+   Through the exercism.org website you can view all your solutions.
+3. **Right of portability:**
+   We can provide you with a copy of your personal data in a machine readable format.
+   Please contact privacy@exercism.org for more information.
+4. **Right to Erasure:**
+   We can remove your personal data from our systems so that you can no longer be personally identified to your solutions.
+   See the section below for details on how to remove your account.
+
+It is important to note that any content you upload to the site (including, but not limited to, solutions to exercises, comments, and mentor feedback) is transferred to us under a non-revocable license and therefore cannot be deleted from the Platform.
+
+## Deleting your data
+
+To delete your data, please log into your account, visit the [Settings page](https://exercism.org/settings) and click "Delete Account".
+
+### Deleting data via email
+
+We take the risk of unintential data loss very seriously, and actively protect against malicious attempts to delete users data through email impersonation.
+The ICO states that organisations "have a legal responsibility to identify that an individual has made a request to you and handle it accordingly" and "if you have doubts about the identity of the person making the request you can ask for more information".
```

Comment:
What's the ICO?

Reply:
https://ico.org.uk/

I'll expand the acronym - thanks.